libs/libcommon/include/common/find_first_symbols.h

```diff
@@ -144,3 +144,9 @@ inline const char * find_first_symbols(const char * begin, const char * end)
 #endif
         return detail::find_first_symbols_sse2<symbols...>(begin, end);
 }
+
+template <char... symbols>
+inline char * find_first_symbols(char * begin, char * end)
+{
+    return const_cast<char *>(find_first_symbols<symbols...>(const_cast<const char*>(begin), const_cast<const char*>(end)));
```

Comment:
I guess, that second and third const_cast are not required.

Reply:
It'll leads to recursive calls. A simple test case
```
const char * foo(const char * x)
{
    return "";
}

char * foo(char * x)
{
    return const_cast<char *>(foo(x));
}
```

Reply:
Ok. I've made a little modification.